src/engine/websites/DoujinDesu.ts

```diff
@@ -1,22 +1,32 @@
-import { Tags } from '../Tags';
```

Comment:
The whole content of this file is marked as changed, maybe saved in different encoding?

I suggest to not fix the website functionality itself within this PR, as it is a different scope => separate PR.
Only update the new parameter for `Common.MangasMultiPageCSS`.

In case of a follow-up PR for DoujinDesu i recommend to get the images from the `/themes/ajax/ch.php` endpoint
```sh
curl 'https://212.32.226.234/themes/ajax/ch.php' \
  -H 'content-type: application/x-www-form-urlencoded; charset=UTF-8' \
  -H 'cookie: PHPSESSID=qtbaqp0k6tcnc97bp3m5824hp7' \
  -H 'origin: https://212.32.226.234' \
  -H 'referer: https://212.32.226.234/ikusa-otome-to-ikusa-goto-onna-samurai-hen-part3/' \
  -H 'x-requested-with: XMLHttpRequest' \
  --data-raw 'id=21309' 
```

Reply:
ok